lib/tree-view.coffee

```diff
@@ -957,103 +1005,132 @@ class TreeView
     @list.classList.remove('multi-select')
     @list.classList.add('full-menu')
 
-  # Public: Toggle multi-select class on the main list element to display the the
+  # Public: Toggle multi-select class on the main list element to display the
   #         menu with only items that make sense for multi select functionality
   showMultiSelectMenu: ->
     @list.classList.remove('full-menu')
     @list.classList.add('multi-select')
 
+  toggleMultiSelectMenu: ->
```

Comment:
`toggle` generally implies "if on, turn off (and vice versa)".  Therefore a better name for this method might be `showMultiSelectMenuIfNecessary`.